packages/remix-server-runtime/server.ts

```diff
@@ -248,6 +267,7 @@ async function handleDocumentRequestRR(
 
   // Sanitize errors outside of development environments
   if (context.errors) {
+    Object.values(context.errors).forEach((err) => handleError(err));
```

Comment:
This is part (1) of what what we were missing before for logging unsanitized errors on document requests